lib/plenario_web/templates/web/chart_dataset/_info_block.html.eex

```diff
@@ -0,0 +1,9 @@
+<div class="panel panel-default">
```

Comment:
Again with the panels, but also: is there more content planned for this in future? Because having an entire template for a single paragraph of static text seems needlessly complex.

Reply:
I forgot to finish writing this. Good catch.